test/helpers/time_helper.rb

```diff
@@ -16,4 +17,52 @@ def time_travel(val)
     Time.unstub(:now)
     Process.unstub(:clock_gettime)
   end
+
+  private
+
+  def fake_stub_all_clock_gettime_behaviour
```

Comment:
This is a bit verbose. It seems to pass if you omit arguments from the stub, or use a helper and stub that instead.